### PR TITLE
add Player Only Parameter to Creep, Hunt and Rush

### PR DESCRIPTION
**Pull request by jokoho48**
### When merged this pull request will:

1. *Title*

### CODE REVIEW

File: addons/wp/stringtable.xml [hunk 1]
```diff
@@ -172,6 +172,12 @@
         <Key ID="STR_Lambs_WP_Module_TaskCreep_MovingCenter_ToolTip">
             <English>The creeping group will use the group leader as a center for the search pattern. Disable to have the unit use the module position instead. This creates a more defensive group</English>
         </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskCreep_PlayersOnly_DisplayName">
+            <English>Players Only</English>
+        </Key>
+        <Key ID="STR_Lambs_WP_Module_TaskCreep_PlayersOnly_ToolTip">
+            <English>The creeping group will only Use Players as targets.</English>
+        </Key>
```
Comment (diwako) [inline addons/wp/stringtable.xml]: Why not just one translation for this? Means we have for the none tool tip duplicates. Might as well add placeholders and for `format` on the translations

File: addons/wp/functions/fnc_findClosestTarget.sqf [hunk 2]
```diff
@@ -16,27 +16,28 @@
  * Public: No
 */
 
-params ["_group", ["_radius", 500], ["_area", [], [[]]], ["_pos", [], [[]]]];
+params ["_group", ["_radius", 500], ["_area", [], [[]]], ["_pos", [], [[]]], ["_onlyPlayers", true]];
```
Comment (diwako) [inline addons/wp/functions/fnc_findClosestTarget.sqf]: Would set the default value of `_onlyPlayers` to false as it affects more than the modules

Comment (jokoho48) [reply addons/wp/functions/fnc_findClosestTarget.sqf]: No, before Only player were used. now allUnits can be used. to not break backwards compatibility it needs to be true

File: addons/wp/stringtable.xml (hunk 1 above)
Comment (jokoho48) [reply addons/wp/stringtable.xml]: I wanted to keep it the same with all the other modules and later have a pass to remove all duplicates.

**Pull request merged**